moveit_kinematics/test/test_ikfast_plugins.sh

```diff
@@ -6,6 +6,15 @@
 
 # We will create ikfast plugins for fanuc and panda from moveit_resources
 # using the script auto_create_ikfast_moveit_plugin.sh
+if [ "$ROS_DISTRO" == "noetic" ]; then
+	sudo ln -s /usr/bin/python3 /usr/local/bin/python
```

Comment:
Should we use `update-alternatives` here instead?

Reply:
I tried `alternatives` first, but python wasn't configured for `alternatives`. Not sure, what I missed.

Reply:
I pushed another commit for this.